easybuild/easyconfigs/h/Harminv/Harminv-1.4.1-GCC-6.4.0-2.28.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'ConfigureMake'
+
+name = 'Harminv'
+version = '1.4.1'
+
+homepage = 'http://ab-initio.mit.edu/wiki/index.php/Harminv'
+description = """Harminv is a free program (and accompanying library) to 
+ solve the problem of harmonic inversion given a discrete-time, finite-length
+ signal that consists of a sum of finitely-many sinusoids (possibly exponentially
+ decaying) in a given bandwidth, it determines the frequencies, decay constants, 
+ amplitudes, and phases of those sinusoids."""
+
+toolchain = {'name': 'GCC', 'version': '6.4.0-2.28'}
```

Comment:
This is not with GCCcore like the rest? Or why not directly with foss?

Reply:
@wpoely86 You cannot build it with `GCCcore` since it requires `OpenBLAS`. I have the tendency of lowering the toolchain level whenever I can and in this case it seemed relatively harmless to explicitly add the `OpenBLAS` dependency. But if we prefer it with the `foss` toolchain, I can change it.